packages/mui-base/src/Select/Select.test.tsx

```diff
@@ -1265,82 +1265,6 @@ describe('<Select />', () => {
     expect(selectButton).to.have.text('1, 2');
   });
 
-  it('perf: does not rerender options unnecessarily', async () => {
```

Comment:
This test appears to cause unrelated tests for `NumberInput` to fail; specifically, the `largeStep` ones that listen on the window keydown. 

Since it's only a perf test, it seems safe to delete, although maybe someone else has a better suggestion.

Reply:
I wouldn't delete it. It seems that there's a bigger issue with tests affecting each other and we need to figure out what's going on. I'm going to investigate it as well.

Reply:
For the time being, not to block your PR, you can skip this test when running on JSDOM. This way, it'll still run when tested with Karma.

Reply:
I found out that we can isolate the tests by adding the `--parallel` flag to mocha (this can also speed them up). However, I haven't gotten to the root cause of these issues yet.